backend/app/views/spree/admin/products/_form.html.erb

```diff
@@ -36,6 +36,13 @@
           <% end %>
         </div>
 
+        <div data-hook="admin_product_form_primary_taxons">
+          <%= f.field_container :primary_taxon do %>
+            <%= f.label :primary_taxon_id, "Primary Taxon" %><br>
```

Comment:
Please let Rails handle the translation and do not hard code English words into templates like this.

This attribute is translated like this

```yaml
en:
  activerecord:
    attributes:
      spree/product:
        primary_taxon_id: Primary Taxon
        primary_taxon: Primary Taxon
```

```suggestion
            <%= f.label :primary_taxon_id %><br>
```